qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/SnapshotReplicationTest.java

```diff
@@ -77,9 +77,16 @@ public void shouldReplicateSnapshots() {
   public void shouldReceiveLatestSnapshotOnRejoin() {
     // given
     final var leaderNodeId = clusteringRule.getLeaderForPartition(1).getNodeId();
-    final var otherBrokers = clusteringRule.getOtherBrokerObjects(leaderNodeId);
-    final var firstFollowerId = otherBrokers.get(0).getConfig().getCluster().getNodeId();
-    final var secondFollowerId = otherBrokers.get(1).getConfig().getCluster().getNodeId();
+    final var followers =
+        clusteringRule.getOtherBrokerObjects(leaderNodeId).stream()
+            .map(b -> b.getConfig().getCluster().getNodeId())
```

Comment:
Nit: I guess we could just sort by nodeId and always pick the last one, which is for sure not 0. But the intent of what we're doing may be less clear as we're not explicitly checking for `!= 0`, which is more obvious. Plus I hope this is temporary, so :man_shrugging: